crates/zoon/src/style/rounded_corners.rs

```diff
@@ -187,7 +186,7 @@ fn compute_radii(corners: RoundedCorners, width: f64, height: f64) -> String {
         // left side & adjacent radii
         height / (radii[3] + radii[0]),
     ];
-    let smallest_ratio = array::IntoIter::new(ratios).fold(f64::INFINITY, |a, b| a.min(b));
+    let smallest_ratio = IntoIterator::into_iter(ratios).fold(f64::INFINITY, |a, b| a.min(b));
```

Comment:
Can we call `ratios.into_iter().fold...` instead?
(The same question applies to other cases, too)

Reply:
So the new code should looks like this ? 

```rust

    let smallest_ratio = IntoIterator::into_iter(ratios.fold(f64::INFINITY, |a, b| a.min(b)));
    
```

What is the difference with what we have ? Is it a question of aesthetic or is there a technicality behind ?

Reply:
I got errors when I try to do it actually.

Reply:
`let smallest_ratio = ratios.into_iter().fold(f64::INFINITY, |a, b| a.min(b)));`

`array::IntoIter::new` was a temporary workaround for `arrays` because it wasn't possible to implement a trait for them before implementing support for const generics in Rust if I'm not wrong - see note in https://doc.rust-lang.org/stable/std/array/struct.IntoIter.html#method.new and I'm sure there were some related blog posts.

`IntoIterator::into_iter(..)` should be just an alternative to `.into_iter()`. I think `IntoIterator` has been implemented for `arrays` in the latest Rust Edition. Calling trait methods through the trait type makes sense when there are conflicts - e.g. multiple traits with the method `into_iter` would be implemented for `array` and `use`d in the current scope.

Reply:
I think I updated all of them correctly. Now I understand :smile: 
Thx for the explanations a lot.